src/app/models/AudioRecording.ts

```diff
@@ -33,44 +48,63 @@ export class AudioRecording extends AbstractModel
   public readonly kind: "AudioRecording";
   public readonly id?: Id;
   public readonly uuid?: Uuid;
+  public readonly uploaderId?: Id;
   public readonly recordedDate?: DateTimeTimezone;
   public readonly siteId?: Id;
-  public readonly durationSeconds?: number;
+  public readonly durationSeconds?: Duration;
```

Comment:
this seems incorrect.

It'd just be a `duration` if it is no longer seconds. Add a getter maybe